Makefile

```diff
@@ -88,7 +89,7 @@ shell: _requires-sandbox-up  # Drop into a development shell
 .PHONY: register
 register: _requires-sandbox-up  ## Register Flyte cookbook workflows
 	$(call LOG,Registering example workflows in cookbook/$(EXAMPLES_MODULE))
-	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) register)
+	@cookbook/scripts/register-all-examples.sh
```

Comment:
The old register used to register the current codebase ... the new script seems to ignore the current/local code and always registers what's pushed to github release if I'm understanding this correctly....

Reply:
as per slack coversation we need two setup : 
```
1. When you run make start , it registers the current tip of master.... this should now use flytectl and run your register-all-examples script...
2. When you run make register, it builds a new image and registers current/local code... this should continue to do so but use flytectl instead...
```